backend/serverless.yml

```diff
@@ -40,17 +40,38 @@ resources:
     appReviewsTable:
       Type: AWS::DynamoDB::Table
       Properties:
-        TableName: appReviewsTable-${self:provider.stage}
+        TableName: reviewsTable-${self:provider.stage}
```

Comment:
We shouldn't need to keep changing the name here - just go into the AWS Console and manually delete the old table. (Or issue `sls remove -s <stage>` and then redeploy)